dropwizard-client-metrics/src/main/scala/org/http4s/client/metrics/codahale/CodaHaleOps.scala

```diff
@@ -0,0 +1,74 @@
+package org.http4s.client.metrics.codahale
```

Comment:
I thought the project had moved away from the "Coda Hale" branding at Coda's request.  Isn't it all in a dropwizard package these days?  I started using Prometheus more, so I haven't kept up.

Reply:
The code is still under codahale package, but the project is effectively called Dropwizard Metrics. I'll change that!